cfgov/jinja2/v1/_includes/templates/content-sidebar.html

```diff
@@ -0,0 +1,19 @@
+{% for block in page.sidefoot %}
```

Comment:
since we've moved this code to its own template.

We'll need to include any imports it depends on like. for now it only looks like:
`{% import 'molecules/related-posts.html' as related_posts with context %}` is necessary